### PR TITLE
Datagrid aggregate position

### LINKED ISSUE #3246: Do you have setting for showing DataGridAggregate row on the top of table?
Do you have setting for showing DataGridAggregate row on the top of table?

It would be very helpful if DataGrid has setting for showing DataGridAggregate row on the top of table (after header row)

And can you add setting css class to DataGridAggregate component ?

### PR DESCRIPTION
**Pull request by David-Moreira**
Closes #3246

### CODE REVIEW

File: Source/Extensions/Blazorise.DataGrid/DataGrid.razor [hunk 1]
```diff
@@ -117,6 +117,10 @@
                         }
                     </TableRow>
                 }
+                @if ( HasAggregates && !Virtualize && ( AggregateRowPosition == DataGridAggregateRowPosition.Top || AggregateRowPosition == DataGridAggregateRowPosition.TopAndBottom ) )
```
Comment (stsrki) [inline Source/Extensions/Blazorise.DataGrid/DataGrid.razor:120]: Maybe also have a property getter to check to for AggregateRowPosition as we did for most other stuff? 

eg. `bool AggregateOnTop => AggregateRowPosition == DataGridAggregateRowPosition.Top || AggregateRowPosition == DataGridAggregateRowPosition.TopAndBottom;`

File: Source/Extensions/Blazorise.DataGrid/DataGrid.razor [hunk 2]
```diff
@@ -178,10 +182,10 @@
                     </_DataGridFullColumnSpanRow>
                 }
             </TableBody>
-            @if ( HasAggregates && !Virtualize )
+            @if ( HasAggregates && !Virtualize && ( AggregateRowPosition == DataGridAggregateRowPosition.Bottom || AggregateRowPosition == DataGridAggregateRowPosition.TopAndBottom ) )
```
Comment (stsrki) [inline Source/Extensions/Blazorise.DataGrid/DataGrid.razor:185]: Same as for top aggregate.

File: Blazorise.slnf [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (stsrki) [inline Blazorise.slnf:1]: Do we really need to have `.slnf` files? Maybe add them to gitignore and just leave them locally?

Comment (David-Moreira) [reply Blazorise.slnf:1]: We don't if you don't like them.

**Review by stsrki — APPROVED** [review]
LGTM

**Pull request merged**